src/Plugin/MediaEntity/Type/EmbeddableVideo.php

```diff
@@ -155,7 +152,7 @@ public function getField(MediaInterface $media, $name) {
    *   URI of local copy of thumbnail.
    */
   protected function localThumbURI(VideoProviderInterface $provider) {
-    return $this->configFactory->get('pub_media.settings')->get('local_images') . '/' . $provider->id() . '_' . $provider->videoId() . '.jpg';
+    return $this->configuration->get('local_images') . '/' . $provider->id() . '_' . $provider->videoId() . '.jpg';
```

Comment:
Other provider modules ship configuration in their own namespaces (as does this one). This reads from media_entity namespace, which is wrong (at least with the current approach we're taking).